crates/bevy_render/src/deterministic.rs

```diff
@@ -0,0 +1,14 @@
+use bevy_ecs::system::Resource;
+
+/// Configure deterministic rendering.
```

Comment:
These docs need to mention z-fighting: this is the dominant term others will look for.